samples/template/docker-compose.x86.yml

```diff
@@ -29,6 +29,13 @@ services:
       - --socket
       - pub+connect:ipc:///tmp/zmq-sockets/input-video.ipc
       - --sync
+    deploy:
```

Comment:
Why this resource is required for the module input?

Reply:
I added it to avoid import errors from some plugins, e.g.
```
template-uri-input-1       | Traceback (most recent call last):
template-uri-input-1       |   File "/opt/savant/gst_plugins/python/frame_tag_filter.py", line 5, in <module>
template-uri-input-1       |     import pyds
template-uri-input-1       | ImportError: libcuda.so.1: cannot open shared object file: No such file or directory
template-uri-input-1       | 
template-uri-input-1       | (gst-plugin-scanner:10): GStreamer-WARNING **: 08:43:55.977: Failed to load plugin '/usr/lib/x86_64-linux-gnu/gstreamer-1.0/deepstream/libnvdsgst_infer.so': libcuda.so.1: cannot open shared object file: No such file or directory
```

Reply:
@denisvmedyantsev OK